s3-native-client/src/main/java/com/amazonaws/s3/S3NativeClient.java

```diff
@@ -302,35 +353,33 @@ protected void populatePutObjectRequestHeaders(final Consumer<HttpHeader> header
             headerConsumer.accept(new HttpHeader("x-amz-grant-write-acp", request.grantWriteACP()));
         }
         if (request.serverSideEncryption() != null) {
-            headerConsumer.accept(new HttpHeader("x-amz-server-side-encryption",
-                    request.serverSideEncryption().name()));
+            headerConsumer
+                    .accept(new HttpHeader("x-amz-server-side-encryption", request.serverSideEncryption().name()));
         }
         if (request.storageClass() != null) {
             headerConsumer.accept(new HttpHeader("x-amz-storage-class", request.storageClass().name()));
         }
         if (request.websiteRedirectLocation() != null) {
-            headerConsumer.accept(new HttpHeader("x-amz-website-redirect-location",
-                    request.websiteRedirectLocation()));
+            headerConsumer.accept(new HttpHeader("x-amz-website-redirect-location", request.websiteRedirectLocation()));
         }
         if (request.sSECustomerAlgorithm() != null) {
-            headerConsumer.accept(new HttpHeader("x-amz-server-side-encryption-customer-algorithm",
-                    request.sSECustomerAlgorithm()));
+            headerConsumer.accept(
+                    new HttpHeader("x-amz-server-side-encryption-customer-algorithm", request.sSECustomerAlgorithm()));
         }
         if (request.sSECustomerKey() != null) {
-            headerConsumer.accept(new HttpHeader("x-amz-server-side-encryption-customer-key",
-                    request.sSECustomerKey()));
+            headerConsumer
+                    .accept(new HttpHeader("x-amz-server-side-encryption-customer-key", request.sSECustomerKey()));
         }
         if (request.sSECustomerKeyMD5() != null) {
-            headerConsumer.accept(new HttpHeader("x-amz-server-side-encryption-customer-key-MD5",
-                    request.sSECustomerKeyMD5()));
+            headerConsumer.accept(
+                    new HttpHeader("x-amz-server-side-encryption-customer-key-MD5", request.sSECustomerKeyMD5()));
         }
         if (request.sSEKMSKeyId() != null) {
-            headerConsumer.accept(new HttpHeader("x-amz-server-side-encryption-aws-kms-key-id",
-                    request.sSEKMSKeyId()));
+            headerConsumer.accept(new HttpHeader("x-amz-server-side-encryption-aws-kms-key-id", request.sSEKMSKeyId()));
         }
         if (request.sSEKMSEncryptionContext() != null) {
-            headerConsumer.accept(new HttpHeader("x-amz-server-side-encryption-context",
-                    request.sSEKMSEncryptionContext()));
+            headerConsumer
+                    .accept(new HttpHeader("x-amz-server-side-encryption-context", request.sSEKMSEncryptionContext()));
```

Comment:
These strings should go in constants somewhere.

Reply:
Agreed, there's some house keeping that needs to be done here.  I'd like that to be a separate PR since this code is only changed due to formatting changes.